lib/ukboot/boot.c

```diff
@@ -251,12 +257,30 @@ void ukplat_entry(int argc, char *argv[])
 	}
 
 #ifdef CONFIG_LIBUKLIBPARAM
-	rc = (argc > 1) ? uk_libparam_parse(argv[0], argc - 1, &argv[1]) : 0;
-	if (unlikely(rc < 0))
-		uk_pr_crit("Failed to parse the kernel argument\n");
-	else {
-		kern_args = rc;
-		uk_pr_info("Found %d library args\n", kern_args);
+	/*
+	 * First, we scan if we can find the stop sequence in the kernel
+	 * cmdline. If not, we assume that there are no uklibparam arguments in
+	 * the command line.
+	 * NOTE: argv[0] contians the kernel/program name that we need to
```

Comment:
Saw that the linter didn't catch this one:
s/contians/contains

Reply:
I'll fix it with PR #867. Thanks :+1: